### DISCUSSION TITLE
bun outdated - does this command exist?

### DISCUSSION THREAD

**Discussion by jinsley8** [oven-sh/bun#7739] [Q&A]
I use `npm outdated` or `yarn outdated` in projects to check for outdated dependencies.

Does `bun` have a similar cli command?

**Comment by Jarred-Sumner** [comment]
i wish

**Reply by jinsley8** [reply]
It looks like `npm outdated` still works when I use `bun` as the package manager.

`yarn outdated` is still better than the `npm outdated` as it shows all updates available.

Something similar to `yarn outdated` for `bun` in the future would be cool.
